src/test/java/seedu/address/model/employee/RemarkTest.java

```diff
@@ -27,6 +28,13 @@ public void isValidRemark() {
         assertTrue(Remark.isValidRemark("-")); // one character
     }
 
+    @Test
+    public void hashCodeGenerator_nonEmptyString_isValidHashCode() {
+        String testRemarkString = "test remark";
+        Remark remark = new Remark(testRemarkString);
+        assertEquals(remark.hashCode(), testRemarkString.hashCode());
+    }
+
```

Comment:
this will break when refactoring from remark to project is done

Reply:
Yup, I will add the label as you suggested.

Reply:
Yup, I will add the label as you suggested. Yup the hashcode test will break, we can fix it later as necessary.